CONTRIBUTING.md

```diff
@@ -11,34 +11,112 @@ This document assumes you have followed the [Apache Geode Code contribution inst
 * Submit a pull request
 
 ## Testing
-   Before submitting a pull request the unit and integration tests must all pass. We are using CTest, (Please see [the CTest documentation](https://cmake.org/Wiki/CMake/Testing_With_CTest) for further information.)
+Before submitting a pull request the unit and integration tests must all pass. We are using CTest, (Please see [the CTest documentation](https://cmake.org/Wiki/CMake/Testing_With_CTest) for further information.)
 ### Running unit tests
-    $ cd <clone>
-    $ cd build
+```bash
+$ cd <clone>
+$ cd build
+```
+The following steps will be updated once the "run-unit-tests" target is fixed.
 
-   The following steps will be updated once the "run-unit-tests" target is fixed.
+```bash
+$ cd cppcache/test/<Debug|Release|if needed>
+$ ./gfcppcache_unittests
+```
 
-    $ cd cppcache/test/<Debug|Release|if needed>
-    $ ./gfcppcache_unittests
 ### Running integration tests
-    $ cd <clone>
-    $ cd build
-    $ cmake --build . --target run-integration-tests
+```bash
+$ cd <clone>
+$ cd build
+$ cmake --build . --target run-integration-tests
+```
 
-   Which is equivalent to running ctest directly:
+Which is equivalent to running ctest directly:
 
-    $ cd build/cppcache/integration-test
-    $ ctest --timeout 2000 -L STABLE -C <Debug|Release> -R . -j1
-   This will take ~ 2 hours, YMMV... you can up the jobs to 4 and run in parallel, but you may end up with test failures that will need to be re-run sequentially.  Like so:
+```bash
+$ cd build/cppcache/integration-test
```

Comment:
Is it worth calling out the `clicache/integration-test` explicitly since we should also expect those tests to pass?

Reply:
Should work if running on Windows, otherwise is a no-op